Library/ENV/4.3/cc

```diff
@@ -213,13 +213,13 @@ class Cmd
     elsif path.start_with?(prefix)
       true
     else
-      !path.start_with?("/opt", "/sw", "/usr/X11")
+      !path.start_with?("/opt/local", "/sw", "/usr/X11")
```

Comment:
Is this change a good opportunity to add a comment that mentions where those paths come from? I suspect there is an increasing number of prospective contributors who have never worked with Fink or MacPorts (or haven't even heard of them).